lib/active_merchant/billing/gateways/ebanx.rb

```diff
@@ -147,6 +156,10 @@ def scrub(transcript)
 
       private
 
+      def add_processing_type(post, options)
+        post[:processing_type] = options[:processing_type] if options[:processing_type]
```

Comment:
do you know if processing_type is an actual field that Ebanx supports the request bode or is this just for setting the header? If it's just for the header then I would suggest passing options all the way down to the headers method

Reply:
By passing the field like this we are sending it in the request body and not just the header. If this is just a field that is passed in hear then we shouldn't be passing it in the request body